docs/user-guide/configure-zowe-zosmf-workflow.md

```diff
@@ -10,175 +13,104 @@ Ensure that you meet the following requirements before you start your Zowe confi
-
-The **perform workflows** step enables you to run either all attached workflows or just the mandatory one — the post-deployment workflow for mounting.
-
-### Execute workflow from software instance
-
-Software instance is created after PSWI deployment is complete. Execute a workflow from an SI.
-
-**Follow these steps**:
-
-1. Log in to z/OSMF.
-2. Select the **Software Management** panel.
-3. In the displayed table, select **Software Instances**.
-4. Select the checkbox next to the **Software Instance Name** column for the instance you want to execute the workflow against.
-5. Select the **Perform Workflows** option from the **Actions** menu.
+![Deployment Checklist](../images/zosmf/perform-workflows.png)
 
-   The **Software Management Software Instances Perform Workflows** dialog opens.
+The **Perform Workflows** step enables you to run either all attached workflows or just the
+post-deployment workflow for mounting, which is required.
 
-6. Select the **Create Workflow** option from the **Actions** menu.
-7. In the displayed table, click on the name of the workflow you want to execute.
-8. Click **OK**.
+After you successfully performed the Zowe Mount workflow, you can start the **Configuration of Zowe 3.0** workflow.
+When you select this workflow from the list of Workflow Definition Files, the following screen displays: 
 
-   The **Workflows** tab with the previously selected workflow opens.
+![Configuration of Zowe 3.0 workflow](../images/zosmf/workflow-zoweConfiguration.png)
 
-9. Execute the workflow steps.
+You can see the workflow details by expanding the Workflow details panel.
+This workflow has three main steps:
 
-You have successfully executed a workflow from a software instance.
+### 1. **Define variables**
 
-## Register and execute workflow in the z/OSMF web interface
+This workflow step includes the list of Zowe variables and contains many child sub-steps.
+When you expand this step, the following screen displays: 
 
-z/OSMF workflow simplifies the procedure to configure and start Zowe. Execute the following steps to register and execute the workflow in the z/OSMF web interface:
+![Step1: Define variables](../images/zosmf/workflow-defineVariables.png)
 
-1. Log in to the z/OSMF web interface and select **Use Desktop Interface**.
+First, define the main variables for your configuration.   
+The other sub-steps are optional depending on which components you choose to enable.
 
-2. Select the Workflows File.
+Perform the following steps to execute each sub-step individually:
 
-3. Select **Create Workflow** from the **Actions** menu.
+1. Click the title of the sub-step.
+2. Select the **Perform** tab.
+3. Review the step contents and update the variables that are marked by a red asterisk based on your mainframe environment.
 
-   The **Create Workflow** panel appears.
+   On the components page of **Define the main variables** sub-step, select the components that you wish to enable. 
```

Comment:
Should this be a separate step after #3 Review the step contents? It seems that this is a separate step after the review...

Reply:
I was thinking about it, but this is a part of point 3 for only particular su-step (Define the main variable).